main.c

```diff
@@ -613,7 +635,7 @@ static void handle_global(void *data, struct wl_registry *registry,
 		create_output(state, wl_output);
 	} else if (strcmp(interface, zxdg_output_manager_v1_interface.name) == 0) {
 		state->xdg_output_manager = wl_registry_bind(registry, name,
-			&zxdg_output_manager_v1_interface, 1);
+			&zxdg_output_manager_v1_interface, 2);
```

Comment:
This may break if the compositor only supports version 1. If that is a concern, I think `MIN(2, version)` would be better.

Reply:
I think the `-o` feature w/o outputs names is kinda useless. But that is a question to @emersion.

Reply:
I think it's fine to bump the minimum required version to 2.